roles/galaxy_media_site/tasks/main.yml

```diff
@@ -1,5 +1,9 @@
 ---
 
+- name: Ensure PostgreSQL is restarted immediately after configuration change
+  meta: flush_handlers
+  tags: postgresql
```

Comment:
`flush_handlers` only executes handlers that have been notified up to this point. So this should be added _after_ the changes, where you originally had the `restart postgresql` task

Reply:
see: [ansible.builtin.meta module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/meta_module.html)

Reply:
Yes, the idea is that it flushes handlers which were notified in the postgresql role which runs before this.